source/opaque.d.ts

```diff
@@ -72,3 +72,36 @@ type Person = {
 @category Type
 */
 export type Opaque<Type, Token = unknown> = Type & Tagged<Token>;
+
+/**
+Remove readonly [tag] from an Opaque type, essentially reverting it back to a generic (i.e. non-Opaque) type.
```

Comment:
I would start this sentence with **what** it does, and then explain in another sentence how it does it.

> Reverts an `Opaque` type back to its original type. [...]

Reply:
Done.